fullstop-plugins/fullstop-application-masterdata-plugin/src/main/java/org/zalando/stups/fullstop/plugin/config/ApplicationMasterdataPluginAutoConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (C) 2015 Zalando SE (http://tech.zalando.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zalando.stups.fullstop.plugin.config;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+import org.zalando.stups.fullstop.plugin.ApplicationMasterdataPlugin;
+
+/**
+ * @author jbellmann
+ */
+@Configuration
+@ComponentScan(basePackageClasses = { ApplicationMasterdataPlugin.class })
+@EnableConfigurationProperties({ ApplicationMasterdataPluginProperties.class })
+public class ApplicationMasterdataPluginAutoConfiguration {
+}
```

Comment:
works the spring context? are you missing here the UserDataProvider(https://github.com/zalando-stups/fullstop/blob/%23144-application-info-incomplete/fullstop-plugins/fullstop-application-masterdata-plugin/src/main/java/org/zalando/stups/fullstop/plugin/ApplicationMasterdataPlugin.java#L68)?

What we configured by the RegistryPluginAutoConfiguration:
https://github.com/zalando-stups/fullstop/blob/master/fullstop-plugins/fullstop-registry-plugin/src/main/java/org/zalando/stups/fullstop/plugin/config/RegistryPluginAutoConfiguration.java#L40